refs/csswg.json

```diff
@@ -94,10 +94,7 @@
         "href": "https://www.w3.org/International/articles/typography/justification"
     },
     "OPEN-FONT-FORMAT": {
-        "source": "https://drafts.csswg.org/biblio.ref",
-        "title": "Information technology — Coding of audio-visual objects — Part 22: Open Font Format",
-        "publisher": "International Organization for Standardization.",
-        "href": "http://standards.iso.org/ittf/PubliclyAvailableStandards/c052136_ISO_IEC_14496-22_2009%28E%29.zip"
+        "aliasOf": "iso14496-22-2019"
```

Comment:
Shouldn't `scripts/csswg.js` be disabled first? I suspect all will go fine but the script might be tempted to override this update next time it runs with the info it still finds in the outdated CSS WG source.

Reply:
I originally thought the same, but then I tested locally and it doesn't happen. Looking at the script:

https://github.com/tobie/specref/blob/5e7c635cc904fed15bbc746996c1ca9ef1cfa4a3/scripts/csswg.js#L66

it appears it's because I removed the `"source"` attribute for OPEN-FONT-FORMAT as part of this that it is not overwritten